src/gfn/env.py

```diff
@@ -379,10 +379,22 @@ def __init__(
         ):
             import warnings
 
+            expert_parameters_used = []
+            if dummy_action is not None:
+                expert_parameters_used.append("dummy_action")
+            if exit_action is not None:
+                expert_parameters_used.append("exit_action")
+            if sf is not None:
+                expert_parameters_used.append("sf")
+            if preprocessor is not None:
+                expert_parameters_used.append("preprocessor")
+
             warnings.warn(
-                "You're using advanced parameters (dummy_action/exit_action/sf/preprocessor). "
+                "You're using advanced parameters: ({}). "
                 "These are only needed for custom action handling. "
-                "For basic environments, you can omit these.",
+                "For basic environments, you can omit these.".format(
+                    ", ".join(expert_parameters_used)
+                ),
```

Comment:
this was here before, but wouldn't this raise a warning even for our environments?

For example:
https://github.com/GFNOrg/torchgfn/blob/73549a05525769ac0f614ce65e2fa903b5692082/src/gfn/gym/box.py#L39-L45

Reply:
yes but that's ok, it's just stating that you're doing something a bit expert level.